dist/source/repo-creator/choose-repository.ts

```diff
@@ -121,13 +121,22 @@ export class ChooseRepository {
 	}
 	
 	protected search = () => {
-		this.clearSearchResults();
+		/*this.clearSearchResults();
 		this.gitHub.search(this.searchInput).then(searchResults => {
 			let resultTemplates = underscore(searchResults).map(searchResult => new Repository(searchResult.owner.login, searchResult.name, false, false, false, true));
 			this.mergeTemplates(resultTemplates);
 		}).catch((error: Error) => {
 			this.eventAggregator.publish(error);
-		});
+		});*/
+		this.gitHub.search(this.searchInput).then(searchResults => {
+    let resultTemplates = underscore(searchResults).map(searchResult => new Repository(searchResult.owner.login, searchResult.name, false, false, false, true));
+    this.mergeTemplates(resultTemplates);
+}).catch((error: Error) => {
+    this.eventAggregator.publish(error);
+}).then(x => {
+    this.searching = false;
+});
+this.searching = true;
```

Comment:
Can you fix this up a bit?  I prefer not to leave commented code in.  Also, the indentation is off here.